src/vorta/assets/UI/mainwindow.ui

```diff
@@ -211,65 +181,49 @@
        </widget>
       </item>
       <item row="0" column="1">
-       <widget class="QProgressBar" name="createProgress">
-        <property name="enabled">
-         <bool>true</bool>
-        </property>
-        <property name="palette">
-         <palette>
-          <active>
-           <colorrole role="Highlight">
-            <brush brushstyle="SolidPattern">
-             <color alpha="255">
-              <red>255</red>
-              <green>151</green>
-              <blue>41</blue>
-             </color>
-            </brush>
-           </colorrole>
-          </active>
-          <inactive>
-           <colorrole role="Highlight">
-            <brush brushstyle="SolidPattern">
-             <color alpha="255">
-              <red>255</red>
-              <green>151</green>
-              <blue>41</blue>
-             </color>
-            </brush>
-           </colorrole>
-          </inactive>
-          <disabled>
-           <colorrole role="Highlight">
-            <brush brushstyle="SolidPattern">
-             <color alpha="255">
-              <red>220</red>
-              <green>220</green>
-              <blue>220</blue>
-             </color>
-            </brush>
-           </colorrole>
-          </disabled>
-         </palette>
-        </property>
-        <property name="value">
-         <number>0</number>
+       <widget class="QLabel" name="progressText">
+        <property name="sizePolicy">
+         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
+          <horstretch>0</horstretch>
+          <verstretch>0</verstretch>
+         </sizepolicy>
         </property>
-        <property name="textVisible">
-         <bool>true</bool>
+        <property name="font">
+         <font>
+          <pointsize>11</pointsize>
+          <weight>75</weight>
+          <bold>true</bold>
```

Comment:
```suggestion
```

Reply:
I actually wanted to make it bold. You think it's a bad idea?

Reply:
You mean the issue is that sometimes the font isn't bold? That wasn't intended.

Reply:
Removed all bold formatting.